src/index.js

```diff
@@ -1,118 +1,23 @@
-import { createInterface } from 'readline';
+import GameLogic from "./domain/gameLogic.js";
+import GameView from "./view/gameView.js";
+import {rl, singlePrompt,twoNumbersPrompt } from './view/prompt.js';
 
-const rl = createInterface({
-  input: process.stdin,
-  output: process.stdout,
-});
 
-const validateInput = (answer) => {
-  if (!answer.includes(',')) {
-    console.log('양식에 맞게 숫자를 입력해주세요 (예: 1,50)');
-    return false;
-  }
-  const numbers = answer.split(',').map(num => parseInt(num, 10));
-  if (numbers.length !== 2 || numbers.some(isNaN)) {
-    console.log('숫자 두 개를 정확히 입력해주세요 (예: 1,50)');
-    return false;
-  }
-  return numbers;
-};
-
-const singlePrompt = (query) => new Promise((resolve) => rl.question(query, resolve));
+async function startGame() {
+  const [min, max] = await twoNumbersPrompt('게임을 시작하기 위해 최소 값, 최대 값을 입력해주세요. (예: 1,50): ');
```

Comment:
p4: 동작에 관련한 함수라면, 동사가 먼저 오는게 적합해요 의도를 명확하게 다른 사람들에게 전달할 수 있어요.
그리고 prompt라는 단어가 명령프롬프트처럼 입력을 요구한다는 것은 알겠으나, get* 이나 input*이 더 직관적이라는 생각이드네요 👍 
```suggestion
  const [min, max] = await inputTwoNumbers('게임을 시작하기 위해 최소 값, 최대 값을 입력해주세요. (예: 1,50): ');
```

Reply:
변경완료했습니다!